_posts/2020-01-01-2019-summary.md

```diff
@@ -0,0 +1,230 @@
+* Support for higher-order [`reduce_agg`](/docs/current/functions/aggregate.html#reduce_agg) 
+  function in aggregation queries.
+* New datetime functions: [`last_day_of_month`](/docs/current/functions/datetime.html#last_day_of_month),
+  [`at_timezone`](/docs/current/functions/datetime.html#at_timezone) and 
+  [`with_timezone`](/docs/current/functions/datetime.html#with_timezone).
+
+## Security
+
+* [Role-based access control](/docs/current/sql/create-role.html) and related commands.
+* [INVOKER security mode](/docs/current/sql/create-view.html#security) for views, which allows views to be run using the permissions of the 
+  current user.
+* Prevent replay attacks and result hijacking in client APIs.
+* JWT-based [internal communication](/docs/current/security/internal-communication.html#internal-authentication) authentication,
+  which obsoletes the need to use Kerberos or certificates and greatly simplifies secure setups.
+* Credential passthrough, which allows Presto to authenticate with the underlying data source with 
+  credentials provided by the user running a query. We found this especially useful when dealing with
+  Google Storage in GCP or SQL databases that manage user authentication and authorization on 
+  their own.
+* Impersonation for [Hive metastore](/docs/current/connector/hive.html#hive-thrift-metastore-configuration-properties).
+* Support for reading and writing encrypted files in HDFS using Hadoop KMS.
+
+## Geospatial
+
+* New geospatial functions: 
+  [`ST_Points`](/docs/current/functions/geospatial.html#ST_Points), 
+  [`ST_Length`](/docs/current/functions/geospatial.html#ST_Length), 
+  [`ST_Area`](/docs/current/functions/geospatial.html#ST_Area), 
+  [`line_interpolate_point`](/docs/current/functions/geospatial.html#line_interpolate_point) and 
+  [`line_interpolate_points`](/docs/current/functions/geospatial.html#line_interpolate_points).
+* `SphericalGeography` type and [related functions](/docs/current/functions/geospatial.html#to_spherical_geography) 
+  to support spatial features in geographic coordinates (latitude / longitude) on spherical model of the earth.
+* Support for Google Maps Polyline format via [`to_encoded_polyline`](/docs/current/functions/geospatial.html#to_encoded_polyline)
+  and [`from_encoded_polyline`](/docs/current/functions/geospatial.html#from_encoded_polyline) functions.
+* [`geometry_from_hadoop_shape`](/docs/current/functions/geospatial.html#geometry_from_hadoop_shape) to decode geometry objects in 
+  Spatial Framework for Hadoop representation
+
+## Cloud Integration
+
+* Support for Azure Data Lake Blob and ADLS Gen2 storage.
+* Support for [Google Cloud Storage](/docs/current/connector/hive-gcs-tutorial.html).
+* Several [performance improvements](https://github.com/prestosql/presto/pull/142) for AWS S3.
+
+## CLI and JDBC Driver
+
+* Syntax highlighting and multi-line editing.
+ 
+  ![](/assets/blog/2019-review/presto-cli.jpeg)
```

Comment:
The image has a shaded gray border around it. Let's trim that off so the image is just the CLI window.